src/carousel.scss

```diff
@@ -132,19 +134,26 @@ $button: #{$fd-namespace}-button;
     box-shadow: none;
     margin: 0.25rem;
 
-    &[class*=sap-icon]::before {
-      display: inline-block;
-      margin: 0;
+    &::before {
+      left: -0.25rem;
+      right: -0.25rem;
+      height: $fd-carousel-touchable-button-size;
+      width: $fd-carousel-touchable-button-size;
     }
 
     @include fd-rtl() {
-      &[class*=sap-icon]::before {
+      & > [class*=sap-icon] {
         transform: rotate(180deg);
       }
     }
 
     @include fd-focus() {
-      outline-offset: 0;
+      &::after {
+        top: $fd-carousel-button-focus-outline-offset;
+        bottom: $fd-carousel-button-focus-outline-offset;
+        left: $fd-carousel-button-focus-outline-offset;
+        right: $fd-carousel-button-focus-outline-offset;
+      }
```

Comment:
Could you change offset to `-3px (in rem ofc)`? Because now it looks like that 
![image](https://user-images.githubusercontent.com/26483208/91832158-591a0f00-ec45-11ea-92a0-cabe4a066dfe.png)

Reply:
With 0 it looks like this:
<img width="244" alt="Screen Shot 2020-09-01 at 9 16 58 AM" src="https://user-images.githubusercontent.com/39598672/91856195-392e1f80-ec34-11ea-8764-1e8ccb4624e8.png">

Changed it to -0.125rem